.env

```diff
@@ -32,6 +32,7 @@ FLAGSMITH_WEBHOOK_SECRET=webhook
 
 INTERNAL_FEED=http://localhost:6000/feed.json?token=token
 
+LINK_PREVIEW_ORIGIN=http://post-scraper-one-ai-server.content.svc.cluster.local
```

Comment:
I think we should add this as a secret pulumi key.
CC: @idoshamun not sure if that's needed?

Reply:
The best practice is to follow Chris's suggestion. We don't add environment variables here even if it's safe to do so. This file is relevant only for local env